tests/foreman/cli/test_remoteexecution.py

```diff
@@ -316,6 +317,75 @@ def test_positive_run_custom_job_template(self):
                 )
             )
 
+    @tier2
+    def test_positive_run_default_job_template_by_ip(self):
+        """Run default template on host connected via ip
+
+        @id: 811c7747-bec6-4a2d-8e5c-b5045d3fbc0d
+
+        @expectedresults: Verify the job was successfully ran against the host
+        """
+        Settings.set({
+            'name': 'remote_execution_connect_by_ip',
+            'value': 'True',
+        })
+        invocation_command = make_job_invocation({
```

Comment:
Maybe we should put next statements into `try...finally` block to return settings value no matter what result we get?